internal/cmd/destinations.go

```diff
@@ -82,16 +87,24 @@ func newDestinationsRemoveCmd() *cobra.Command {
 				return err
 			}
 
+			logging.S.Debug("call server: list destinations named [%s]", args[0])
 			destinations, err := client.ListDestinations(api.ListDestinationsRequest{Name: args[0]})
 			if err != nil {
 				return err
 			}
 
 			if destinations.Count == 0 {
-				return fmt.Errorf("no destinations named %s.", args[0])
+				if force {
+					return nil
+				}
+				return Error{
```

Comment:
We haven't switched all of the other errors over to using this style of Error{} object. Do you want to create this one in the old fmt.Errorf() format and then convert everything over at once? That way we won't have skew between the other styles of errors.